pygeoapi/util.py

```diff
@@ -135,6 +134,23 @@ class EnvVarLoader(yaml.SafeLoader):
     return yaml.load(fh, Loader=EnvVarLoader)
 
 
+def get_api_rules(config: dict) -> config_models.APIRules:
+    """ Extracts the default API design rules from the given configuration.
+
+    :param config:  Current pygeoapi configuration (dictionary).
+    :returns:       An APIRules instance.
+    """
+    rules = config['server'].get('api_rules') or {}
+    rules.setdefault('api_version', __version__)
+    return config_models.APIRules.create(**rules)
+
+
+def get_base_url(config: dict) -> str:
```

Comment:
Added this function so we can get the complete base URL (with optional URL prefix) everywhere using the same code.